media/base/android/java/src/org/chromium/media/MediaPlayerBridge.java

```diff
@@ -78,22 +84,22 @@ protected void destroy() {
         mNativeMediaPlayerBridge = 0;
     }
 
-    protected MediaPlayer getLocalPlayer() {
-        if (mPlayer == null) {
-            mPlayer = new MediaPlayer();
+    protected ExMediaPlayer getExMediaPlayer() {
```

Comment:
Is it better to create `ExMediaPlayerBridge.java` to reduce the rebase workload?